File: 7 Kyu/SplitByValue/split.py
def split_by_value(k, elements):
    returnArr = []
    for i in range(len(elements)):
        if elements[i] < k:
            returnArr.append(elements[i])
    #returnArr.append(k)
    for i in range(len(elements)):
        if elements[i] >= k:
            returnArr.append(elements[i])
    return returnArr

File: 7 Kyu/SplitByValue/split.js
function splitByValue(k, elements){
    let  returnArr = []; 
    for(i = 0; i < elements.length;i++){
      if(elements[i] < k){
        returnArr.push(elements[i]);
      }
    }
    for(i = 0; i < elements.length;i++){
      if(elements[i] >= k){
        returnArr.push(elements[i]);
      }
    }
    return returnArr;
  }